### PR TITLE
Port Model section of Growing guide from Mojo

### PR DESCRIPTION
**Pull request by marcusramberg**
Tried to be more true to the original in this section.

### CODE REVIEW

File: docs/Growing.md [hunk 1]
```diff
@@ -244,6 +244,68 @@ Keep-Alive: timeout=5
...
+  constructor() {
+    this._data = {
+      joel: 'las3rs',
+      marcus: 'lulz',
+      sebastian: 'secr3t'
+    };
+  }
+
+  check(user, pass) {
+    if(!this._data[user]) return false;
```
Comment (kraih) [inline docs/Growing.md:272]: `if(this._data[user] === undefined) ...`

File: docs/Growing.md [hunk 2]
```diff
@@ -244,6 +244,68 @@ Keep-Alive: timeout=5
 Hello World!
 ```
 
+## Model
+
+In mojo.js we consider web applications simple frontends for existing business logic. That means mojo.js
+is by design entirely _model_ layer agnostic, and you just use whatever JavaScript modules you like most.
```
Comment (kraih) [inline docs/Growing.md:250]: Line length seems sketchy. compared to the next section.

File: docs/Growing.md [hunk 3]
```diff
@@ -244,6 +244,68 @@ Keep-Alive: timeout=5
...
+
+``` js
+import mojo from '@mojojs/core';
+import Users from './models/users.js';
+
+export const app = mojo();
+
+app.models.users = new Users();
+
+
```
Comment (kraih) [inline docs/Growing.md:288]: Extra newline.

File: docs/Growing.md [hunk 4]
```diff
@@ -244,6 +244,68 @@ Keep-Alive: timeout=5
...
+app.any('/', async ctx => {
+  // Query parameters
+  const params = await ctx.params();
+  const user = params.get('user')
+  const pass = params.get('pass')
+
+  // Check password
+  if(ctx.models.users.check(user, pass) === true)
+    return ctx.render({text: `Welcome ${user}.`});
```
Comment (kraih) [inline docs/Growing.md:297]: Just use a block if the `if` condition doesn't fit one a single line.

File: docs/Growing.md (hunk 2 above)
Comment (marcusramberg) [reply docs/Growing.md:250]: I just thought it looked more balanced, but altered first line to use full length now :)

File: docs/Growing.md (hunk 4 above)
Comment (marcusramberg) [reply docs/Growing.md:297]: it's less than 120 chars 👍🏻

**Pull request merged**